README.md

```diff
@@ -12,10 +15,12 @@
 npm install -g asyncapi-generator
 ```
 
-Or use all the commands below using Docker by replacing `ag` in all the commands below with:
+Or just use Docker:
 
 ```bash
 docker run --rm -it -v $PWD:/app -w /app asyncapi/generator [COMMAND HERE]
+# Example that you can run inside generator directory after cloning this repository
+docker run --rm -it -v $PWD:/app -w /app asyncapi/generator -o ag -o ./my_output ./test/docs/streetlights.yml markdown
```

Comment:
This does not work in windows 10, neither in PowerShell or Git Bash terminal. We could replace `$PWD` with `${PWD}` which works for PowerShell and pretty sure on Linux systems too 😄

Reply:
I just pushed `{`

do you have Windows and can check? I think for windows you simply need to replace `${PWD}` with `//C/Users/YourUser/`

Reply:
Ill check when I get home :+1:

Reply:
I am getting some weird errors 😄 

**PowerShell**
If I am using the command ` docker run --rm -it -v ${PWD}:/app -w /app gen -o ag -o /my_output ./test/docs/streetlights.yml markdown`  i get the following stacktrace:
```bash
internal/modules/cjs/loader.js:983
  throw err;
  ^

Error: Cannot find module '/app/cli'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```
However when I bind the output path it works fine `docker run --rm -it -v ${PWD}:/app/my_output -w /app gen -o ag -o ./my_output ./test/docs/streetlights.yml markdown`
```
Done! ✨
Check out your shiny new generated files at /app/my_output.
```

**Suggestions**
In the example two output folders are specified, ` -o ag -o ./my_output` for the sake of clarity shouldn't we just include one? 

Second, we bind the app volume meaning the application code will also get included, which I assume we do not want when using docker 🤔 

Third, the workdir argument is not needed since this is already defined in the dockerfile.

So I suggest 
`docker run --rm -it -v ${PWD}:/app/output gen -o /output ./test/docs/streetlights.yml markdown`.
We might change the volume to `-v ${PWD}/output:/app/output` if this is more desired, however in windows the folders have to be manually created.

Reply:
- remove `-w` - I totally agree, removed
- remove `-o ag` - 🤦‍♂ that is actually a stupid copy/paste mistake from my side, sorry for the confusion, removed
- mounting the whole app code - even though it is just an example it should follow best practices, good catch 👌 

After some tweaking, I think now command looks pretty ok for all users.
@jonaslagoni do we need to do something more for windows? I have to admit I got a bit lost in your comment

Reply:
Gonna start a new conversation on the new code.